### ISSUE TITLE
https://github.com/fireeye/capa-rules/issues/289

### ISSUE THREAD

**Issue by Ana06** [fireeye/capa-rules#291]
https://github.com/fireeye/capa-rules/issues/289 shows that we may need to be more strict in PRs to avoid introducing offenses in the nursery rules. Currently `lint.py` allows any kind of warnings for files in the nursery, while I think we only want to allow that examples are missing. Is this right?

**Comment by williballenthin** [comment]
good idea, lets be more strict with lints in the nursery, especially around formatting and the like.
